aptos-move/aptos-vm/src/move_vm_ext/session.rs

```diff
@@ -79,6 +80,9 @@ pub enum SessionId {
         sequence_number: u64,
         script_hash: Vec<u8>,
     },
+    ValidatorTxn {
```

Comment:
Can you please also check `move_vm_ext/vm.rs` file, there is this code in `new_session`:

```
let script_hash = match session_id {
            SessionId::Txn {
                sender: _,
                sequence_number: _,
                script_hash,
            }
            | SessionId::Prologue {
                sender: _,
                sequence_number: _,
                script_hash,
            }
            | SessionId::Epilogue {
                sender: _,
                sequence_number: _,
                script_hash,
            } => script_hash,
            _ => vec![],
        };
```
I recently wanted to change `_` match but did not land that, and since you add a new `SessionId` variant you need to add a pattern there to get the right hash, not an empty vec. The fact that we match like that there is terrible :/

Reply:
good catch! fixed.